tests/unit/insights/test_question.py

```diff
@@ -14,3 +16,1114 @@
+                        },
+                        "200": {
+                            "h": 89,
+                            "w": 200
+                        },
+                        "400": {
+                            "h": 179,
+                            "w": 400
+                        },
+                        "full": {
+                            "h": 894,
+                            "w": 2000
+                        }
+                    },
+                    "white_magic": null
+                },
+                "nutrition_fr": {
+                    "angle": "0",
+                    "coordinates_image_size": "full",
+                    "geometry": "881x427-1501-937",
+                    "imgid": "51",
+                    "normalize": "false",
+                    "rev": "145",
+                    "sizes": {
+                        "100": {
+                            "h": 48,
+                            "w": 100
+                        },
+                        "200": {
+                            "h": 97,
+                            "w": 200
+                        },
+                        "400": {
+                            "h": 194,
+                            "w": 400
+                        },
+                        "full": {
+                            "h": 427,
+                            "w": 881
+                        }
+                    },
+                    "white_magic": "false",
+                    "x1": "1501.3940367879745",
+                    "x2": "2382.8117583069616",
+                    "y1": "937.4230926127373",
+                    "y2": "1364.3598014734966"
+                },
+                "packaging_fr": {
+                    "angle": "0",
+                    "coordinates_image_size": "full",
+                    "geometry": "661x165-220-1100",
+                    "imgid": "49",
+                    "normalize": "false",
+                    "rev": "146",
+                    "sizes": {
+                        "100": {
+                            "h": 25,
+                            "w": 100
+                        },
+                        "200": {
+                            "h": 50,
+                            "w": 200
+                        },
+                        "400": {
+                            "h": 100,
+                            "w": 400
+                        },
+                        "full": {
+                            "h": 165,
+                            "w": 661
+                        }
+                    },
+                    "white_magic": "false",
+                    "x1": "220.58391020569618",
+                    "x2": "881.6472013449367",
+                    "y1": "1100.5069657337815",
+                    "y2": "1265.7727885185916"
+                }
+            }
+        },
+        "status": 1,
+        "status_verbose": "product found"
+    }
+    """
+
+    images = dict(loads(product))
```

Comment:
You shouln't use loads, it's not necessary here. Embed the object as a python dict, or better save it it a JSON file (in test data resources) that you load (as the object is pretty big)